README.md

```diff
@@ -74,94 +74,7 @@ karpenter-5f97c944df-bm85s 1/1   Running 0        15m
 karpenter-5f97c944df-xr9jf 1/1   Running 0        15m
 ```
 
-You can now proceed to deploy the default Karpenter provisioner, and deploy any blueprint you want to test.
-
-#### Deploy a Karpenter Default Provisioner
-
-Before you start deploying a blueprint, you need to have a default [provisioner](https://karpenter.sh/docs/concepts/provisioners/) as some blueprints need it. The Provisioner sets constraints on the nodes that can be created by Karpenter and the pods that can run on those nodes.
-
-Here's the default `Provisioner` and `AWSNodeTemplate` manifest you'll deploy, take your time to explore the configurations of the default provisioner as it's trying to be as flexible as possible:
```

Comment:
Instead of showing this along with the YAML, let's simply reference to the new `.tf ` file (see below comments about this) that includes only the YAML manifest.